assets/styles/themes/_suse_dark.scss

```diff
@@ -0,0 +1,4 @@
+.suse-dark {
+    @include theme-dark($primary: rgb(48, 186, 120));
```

Comment:
I don't understand why the whole mixin and defining/duplicating/loading every possible rule of the theme is needed.  The idea of the themeing is/was:

- Everything theme-able is exposed as a CSS variable
- Related things should refer to an existing variable so they only need to be changed once unless you really want them to be different (e.g. `--thing-border: var(--border)`)
- The "root" theme (which is `light`) is always loaded (and is selected by `::root`)
- Other themes overwrite just what should _differ_ from the root theme (and are selected by `.theme-$name` applied to the body)
- Zero or one of those is selected and applied to the body
- Everything works via plain rendering & CSS selectors, without involving JS, so that it renders right the first time from SSR HTML.

Because of laziness/convenience there is a fair bit of scss-computed calculating colors going on in there now, but rather than doubling down on that with JS implementations we should be getting rid of those by having things that are the primary color use `var(--primary)` instead of `$primary`, etc.

The main use case you're interested in here of changing just the primary color should ideally consist of:

```
.theme-suse-light {
  --primary: rgb(48,186,120);
 ... maybe a few more ...
}
```

And everything else would extend from that.